src/pages/Compare/index.tsx

```diff
@@ -131,6 +144,40 @@ export class CompareBase extends React.Component<Props> {
         }),
       );
     }
+
+    if (
+      version &&
```

Comment:
It looks like this and the following if statement could be wrapped in a check for `version && versionFile &&...` (all common checks).